CHANGELOG.md

```diff
@@ -4,8 +4,9 @@
 
 ### 🦊 What's Changed 🦊
 
- - [`nimbus-cli`](./components/support/nimbus-cli) now has commands: `capture-logs`, `tail-logs`, `test-feature`, `fetch` and `apply-files`. ([#5517](https://github.com/mozilla/application-services/pull/5517)))
- - Add additional Cirrus SDK helper methods and add Python testing for the generated Cirrus Python code [#5478](https://github.com/mozilla/application-services/pull/5478)).
+ - [`nimbus-cli`](./components/support/nimbus-cli) now has commands: `capture-logs`, `tail-logs`, `test-feature`, `fetch` and `apply-files`. ([#5517](https://github.com/mozilla/application-services/pull/5517))
+ - Add additional Cirrus SDK helper methods and add Python testing for the generated Cirrus Python code ([#5478](https://github.com/mozilla/application-services/pull/5478)).
+ - Add `user_id` RandomizationUnit (for Cirrus) ([#5564](https://github.com/mozilla/application-services/pull/5564)).
```

Comment:
just to clarify, why are we adding userId in addition to clientId?

Reply:
`client_id` is a glean-specific term, whereas `user_id` is meant to be more generic and fits the general concept of web applications a little better. @jhugman is that correct? (on the `client_id` piece)